### DISCUSSION TITLE
Incorect reads, for single digit. Ways to correct it

### DISCUSSION THREAD

**Discussion by mztulip** [jomjol/AI-on-the-edge-device#2561] [Q&A] [answered]
Hello,
I have problem with digit reading. Image in my opinion have good quality and contrast. Digits are good. I tested all AI models and it not solve problems.
I want to ask if it is possible to change some parameter to correct it?
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/9120429/cfc05ee4-d714-49fb-9068-88b00e6f35ca)
There is problem with digit 5. Depending on model it can be read as digit 4 or 6 or 1, never as 5. 

Sometimes data looks like showed below. This is for the same configuration. Zero can be interpreted correctly and incorrectly.
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/9120429/3c8a35a6-fd4d-4313-b1ae-af1f75dff134)

**Comment by caco3** [comment, marked as answer]
Deine ROIs sind zu gross. Mach sie etwas kleiner. DU must vermutlich das Ratio ausschalten.

Siehe https://jomjol.github.io/AI-on-the-edge-device-docs/ROI-Configuration/#how-to-setup-the-digit-rois-perfectly

**Comment by mztulip** [comment]
Thanks for link. When I go through documentation I completely missed this part. Now it works significantly better, but not ideally. Sometimes all digits are interpreted incorrectly, but with new photo it is correct.
